tests/test_content_encoding.js

```diff
@@ -13,12 +13,13 @@ test('accepts gzipped content', async t => {
 
   nock('http://example.test')
     .get('/foo')
-    .reply(200, compressed, {
+    .reply('200', compressed, {
```

Comment:
As discussed elsewhere, I don't think we need to add support for this behavior.

If it was there before, throwing an error is probably sufficient.

The exception would be `nock.define` which supports recorded tests. If that currently accepts a string, we should keep that working. Though we can do the cast in `define` instead.